tests/test_span_readable.py

```diff
@@ -0,0 +1,65 @@
+import time
+from instana.span.readable_span import Event, ReadableSpan
+from instana.span_context import SpanContext
+from opentelemetry.trace.status import Status, StatusCode
+
+from instana.w3c_trace_context.traceparent import Traceparent
+
+
+def test_readablespan(
+    span_context: SpanContext,
+    trace_id: int,
+    span_id: int,
+) -> None:
+    span_name = "test-span"
+    timestamp = time.time_ns()
+    span = ReadableSpan(span_name, span_context)
+
+    assert span is not None
+    assert isinstance(span, ReadableSpan)
+    assert span.name == span_name
+
+    context = span.context
+    assert isinstance(context, SpanContext)
+    assert context.trace_id == trace_id
+    assert context.span_id == span_id
+
+    assert span.start_time
+    assert isinstance(span.start_time, int)
+    assert span.start_time > timestamp
+    assert not span.end_time
+    assert not span.attributes
+    assert not span.events
+    assert not span.parent_id
+    assert span.duration == 0
+    assert span.status._status_code == StatusCode.UNSET
```

Comment:
`Status` objects have a property called `status_code`. 
```suggestion
    assert span.status.status_code == StatusCode.UNSET
```

Reply:
Done.